### ISSUE TITLE
SITE062 Usage history duplicated and Increased excess charges added to accounts

### ISSUE THREAD

**Issue by kdjstudios** [kdjstudios/SABillingGitlab#1271]
In GitLab by @kdjstudios on Dec 4, 2018, 09:03

**Submitted by:** "Denise Joseph" <denise.joseph@answernet.com>

**Helpdesk:** http://www.servicedesk.answernet.com/profiles/ticket/2018-12-03-21567

**Server:** Internal

**Client/Site:** Tornonto

**Account:** All

**Issue:** 

The holiday fees was reset.  Billing was reverted and once the service code summary report was pulled it reflected the correct holiday charges for both US and Canadian billing.  The problem that was noticed after billing was finalize is that excess charges was applied to many accounts that did not reflect it prior.  I want to know if there is anyway for us to correct this prior to our next billing cycle? 

Wrote again:

I had asked the below question in Ticket Number:   2018-12-03-21567 with subject line: Re: RE: SITE062 SAB Holiday Charges but it was closed without a response to my question. 

The problem noticed after billing was finalize for billing period 11/24/2018  under (Billing history) the “Excess column” reflected a great increased in excess charges and was applied to many accounts, which was not noticed prior to billing being finalized.  Please see below.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Dec 4, 2018, 09:04

@smasih @amishra @pchaudhary @tim.traylor 

Please look into this and determine the cause.

I wrote:

> Thank you and we have notified the support team. Any excess charges are generally related to concerns with your usage file; not with SA Billing. Please be sure to check your usage file for any errors, duplicates, or discrepancies.  We will also have the support team research and follow up accordingly.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Dec 5, 2018, 08:13

@smasih @amishra @pchaudhary @tim.traylor 

Do we have any updates on this?

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Dec 5, 2018, 09:23

I wrote:

> We have not received any follow up from you on our response below. We are requesting you to attach the original uploaded usage files so the support team can better determine the cause.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Dec 5, 2018, 09:28

Gary wrote:

> WTF - I just saw an email from Elizabeth Fed from winnipeg where customers are complaining.  This is URGENT!!!!

Linked to #1264

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Dec 5, 2018, 10:38

Denise wrote

> Derrick did respond the email below.  However, the email below instructed me to review my usage file for errors, duplicates, or 
> Discrepancies.  I did not realize you wanted me to forward it to you. 
> The original usage file is the first attachment and the second usage file attached is the one you asked me to rerun when the US billing cycle date showed as 02/24/2019 instead of 11/24/2018. Please let me know if you need me to do anything further.
> 
> [November+Billing+Export+_2816_29.csv](/uploads/06b8a64daef87027231e3bdc075eb7b6/November+Billing+Export+_2816_29.csv)
> 
> [November+Billing+Export+_2817_29.csv](/uploads/b076a38029a012ba527018a5d52e1c3b/November+Billing+Export+_2817_29.csv)

I wrote:

> Thank you for the response and the files. Just to clarify the file labeled "November Billing Export (16).csv" is the one ran and uploaded before you reverted. While the file labeled "November Billing Export (17).csv" is the one you ran and uploaded after you reverted. Is that correct?
> 
> We will inform the support team with the updated information and follow up with their response accordingly. 
> 
> NOTE: We did receive a vague secondary ticket from Derrick in which he requested a call to discuss issues. We then had to request which issues he was referring to and have since followed up with those topics accordingly. We need to insure that responses are sent to the appropriate tickets accordingly and that duplicate tickets are not being submitted for the same issue; as that causes more confusion and only delays the process for the support team and tech to find resolutions.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Dec 6, 2018, 08:20

After long communications and discussions yesterday, we were able to put forth a hotfix by having the site revert and rerun their billing. We have further informed the site about checking their billing throughout the process to help reduce these types of errors.

@smasih @amishra @tim.traylor @pchaudhary 

Things we need the Aavaz team to check if possible:
- Tim believes that since the usage file did not contain duplicate values or double usage; that this was a result of the support team not fully removing/deleting the four billing cycles that were created and addressed in #1259

EDIT: This was discussed today, and after reviewing the site's side by side report all appears to be working correctly. We should not need to have the team check further into this and simply can all presume that it was due to the above. Marking for close
